src/components/Dropdown/Dropdown.tsx

```diff
@@ -17,7 +17,10 @@ import { ScreenReader } from '../ScreenReader';
 import { recursivelyMapChildren } from '../utils/recursivelyMapChildren';
 import { DropdownItem, DropdownItemProps, DropdownItemWithIndex } from './DropdownItem';
 import useIsomorphicLayoutEffect from 'use-isomorphic-layout-effect';
-import { useId } from '@reach/auto-id';
+import { TextEncoder} from 'util';
+import { useId } from "react-id-generator";
+
+global.TextEncoder = TextEncoder;
```

Comment:
why do we need this? I'm not seeing it being used in the component

Reply:
when i ran test, i was getting TextEncoder undefined. seems like this was a fix other people used.